azure/rediscaches.bicep

```diff
@@ -40,9 +40,20 @@ param skuFamily string = 'C'
 ])
 param skuName string = 'Basic'
 
+@description('The user-defined tags that will be applied to the resource. Default is null')
+param tags object = {}
+
+@description('The Radius specific tags that will be applied to the resource')
+var radiusTags = {
+  'radapp.io-environment': context.environment.id
+  'radapp.io-application': context.application == null ? '' : context.application.id
+  'radapp.io-resource': context.resource.id
+}
+
```

Comment:
Should this pattern be documented somewhere?

Reply:
added this to the README in /azure directory

Reply:
documented in the readme under azure folder